src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_KeyFigure+Formatting.swift

```diff
@@ -19,13 +19,28 @@ extension SAP_Internal_Stats_KeyFigure {
 				return nil
 			}
 
-			return String(format: AppStrings.Statistics.Card.million, formattedNumber)
+			formattedString = String(format: AppStrings.Statistics.Card.million, formattedNumber)
 		} else {
 			let decimals = max(0, Int(self.decimals))
 			numberFormatter.minimumFractionDigits = Int(decimals)
 			numberFormatter.maximumFractionDigits = Int(decimals)
 
-			return numberFormatter.string(from: NSNumber(value: value))
+			formattedString = numberFormatter.string(from: NSNumber(value: value))
+		}
+		guard let cardId = cardRawValue else {
+			return formattedString
+		}
+		switch HomeStatisticsCard(rawValue: cardId) {
+		case .atLeastOneVaccinatedPerson, .fullyVaccinatedPeople:
+			guard let valueString = formattedString else {
+				return formattedString
+			}
+			return valueString + AppStrings.Statistics.percent
```

Comment:
Would be better to use `numberFormatter.numberStyle = .percent`. Maybe we could just set `formattedValue(inPercent: Bool = false)` and handle the logic which card needs which in the `HomeStatisticsCardViewModel`?